Core.Collectors/Web/HttpExceptionSignature.cs

```diff
@@ -4,21 +4,27 @@
 using Microsoft.CloudMine.Core.Collectors.Collector;
 using System;
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace Microsoft.CloudMine.Core.Collectors.Web
 {
     public class HttpExceptionSignature : IAllowListStatus
     {
-        public static readonly HttpExceptionSignature RequestTimeoutException = new HttpExceptionSignature(exception =>
+        public static HttpExceptionSignature RequestTimeoutException(Func<HttpRequestMessage, List<CollectionNode>> continuationNodeList = null)
```

Comment:
>Func<HttpRequestMessage, List<CollectionNode>> continuationNodeList = null [](http://example.com/codeflow?start=69&length=74)

As usual, can we leave this change (which is not referenced here) to a separate PR?)